.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: /
```

Comment:
Do you need to add the functions folder?

Reply:
couldn't hurt.